src/main/lrsql/util/reaction.clj

```diff
@@ -101,3 +102,19 @@
   [ruleset-bytes]
   (with-open [r (io/reader ruleset-bytes)]
     (cjson/parse-stream r (partial keyword nil))))
+
+(s/fdef generate-statement
+  :args (s/cat :cond-map (s/map-of ::rs/condition-name
+                         ::rs/val)
```

Comment:
`::rs/val` is an open-ended spec covering any JSON. In this case you actually have a good spec to draw on, `:xapi-schema.spec/statement`, as anything coming out of that query should be a valid xAPI statement